change/@fluentui-react-eff13953-3080-4ee9-8ed5-6a778a7f692f.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Panel: Fix async and events to work in concurrent mode",
```

Based on v8's changelog I think this should be:

```suggestion
  "comment": "fix: Panel async and events to work in concurrent mode",
```